single_tech_samples/databricks/sample4_ci_cd/notebook-python-lib/devops/requirements.txt

```diff
@@ -0,0 +1,17 @@
+pip
```

Comment:
Why is this file here?

Reply:
@gary918 Are you using this file anywhere?  If you do, suggest combining with the requirements.txt in the parent folder "notebook-python-lib"

Reply:
@swetha-sundar No. I'm not. I'm Ok for combining them.